src/evict/evict_page.c

```diff
@@ -574,10 +574,14 @@ __evict_review(
 			}
 
 			/*
-			 * Check if reconciliation suggests trying the
-			 * lookaside table.
+			 * We usually only consider lookaside eviction once
+			 * eviction is struggling.  However, eviction in
+			 * service of a checkpoint (which is only done when
+			 * urgent eviction is required) will try lookaside
+			 * eviction immediately.
 			 */
-			if (__wt_cache_aggressive(session) &&
+			if ((__wt_cache_aggressive(session) ||
+			    WT_SESSION_IS_CHECKPOINT(session)) &&
```

Comment:
I'm not sure why we'd want this change. Is there reason to believe the CPU is struggling, what's the additional cost of a second reconciliation attempt? Or are we seeing consistent failure on the first reconciliation?